runTests.py

```diff
@@ -142,7 +142,8 @@ def main():
 
     for batch in batched(tests):
         modelHpp = modelDependencies(batch)
-        makeTest(" ".join(modelHpp), inputs.j)
+        if len(modelHpp) > 0:
+            makeTest(" ".join(modelHpp), inputs.j)
```

Comment:
This is also minor cleanup to not run this make step if no model was needed in this batch of tests.